### ISSUE TITLE
Donasi Sekarang buttons don't have sufficient color contrast ratio

### ISSUE THREAD

**Issue by mazipan** [kawalcovid19/wargabantuwarga.com#666]
## Description

Ask the Designer, can we replace the color using our primary button?

This is the [Lighthouse Report](https://lighthouse-dot-webdotdevsite.appspot.com//lh/html?url=https%3A%2F%2Fwww.wargabantuwarga.com#accessibility).

![Screen Shot 2021-08-12 at 22 00 44](https://user-images.githubusercontent.com/7221389/129220138-e1a18cd3-4092-4a66-9fe7-23dab50773f9.png)

## Tasks

- [ ] ...
- [ ] ...
- [ ] ...

**Comment by zainfathoni** [comment]
[Question has been asked](https://kawalcovid19.slack.com/archives/C0279E2Q99Q/p1628834013026100), let's wait for the response. 😁 
<img width="403" alt="Screenshot 2021-08-13 at 13 54 03" src="https://user-images.githubusercontent.com/6315466/129311314-f814c6e7-3b82-4d9a-ac41-176ddc4fb2b0.png">

**Comment by zainfathoni** [comment]
> Ask the Designer, can we replace the color using our primary button?

Yes, we can. So for whoever taking this ticket, please replace that button with our primary button.
